src/libs/ck-libs/ampi/ampi.C

```diff
@@ -989,6 +991,18 @@ static void ampiNodeInit() noexcept
       CkPrintf("AMPI> Synchronous messaging threshold is %d Bytes.\n", AMPI_SSEND_THRESHOLD);
     }
   }
+  if ((value = getenv("AMPI_MSG_POOL_SIZE"))) {
+    AMPI_MSG_POOL_SIZE = atoi(value);
+    if (CkMyNode() == 0) {
+      CkPrintf("AMPI> Message pool size is %d Bytes.\n", AMPI_MSG_POOL_SIZE);
```

Comment:
Is the unit 'bytes' here? The comment below seems to suggest it is number of messages

Reply:
Fixed